src/hooks/useToggle.test.ts

```diff
@@ -0,0 +1,26 @@
+import { useToggle } from './useToggle';
+import { renderHook, act } from '@testing-library/react-hooks';
+
+test('Should initialize with default value', () => {
```

Comment:
If you have more than one test better to wrap it into "describe":
`
describe('useToggle', () => {
  test('should ...')

 test('should...')
})
`